upstart/slack.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=service handling slack connection
+Wants=network-online.target
+After=network.target network-online.target NetworkManager.service NetworkManager-wait-online.service
+
+[Service]
+Type=simple
+RemainAfterExit=no
+ExecStart=/bin/bash -c "/usr/bin/tmux -2 new-session -d -s slack && \
+    /usr/bin/tmux -2 send -t slack 'python `rospack find slack`/scripts/ros_service.py'
```

Comment:
Command Line params missing for der ros_service.py. They need to be: `token`, `channel`, `username`. We should use placeholders e.g. `my_token`, `my_channel`. The placeholders should then be replaced by StickInstall, Postinstall or Userinstall